issue.go

```diff
@@ -1236,9 +1237,17 @@ func (s *IssueService) GetWatchers(issueID string) (*[]User, *Response, error) {
 	result := []User{}
 	user := new(User)
 	for _, watcher := range watches.Watchers {
-		user, resp, err = s.client.User.Get(watcher.Name)
-		if err != nil {
-			return nil, resp, NewJiraError(resp, err)
+		if watcher.AccountID != "" {
+			// try fallback deprecated method
```

Comment:
I think this comment is in the wrong place, since `GetByAccountID` is not the deprecated method.

Reply:
Fixed